python/ray/data/_internal/stats.py

```diff
@@ -227,8 +227,16 @@ def TODO():
         """Placeholder for ops not yet instrumented."""
         return DatasetStats(stages={"TODO": []}, parent=None)
 
-    def summary_string(self, already_printed: Set[str] = None) -> str:
-        """Return a human-readable summary of this Dataset's stats."""
+    def summary_string(
+        self, already_printed: Set[str] = None, log_parent: bool = True
```

Comment:
shall we just call it `print_parent` or `include_parent` to be consisten with wording `already_printed`?